config/prod/dian-prod-synapse-import-s3.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.j2"
+stack_name: "dian-prod-synapse-import-s3"
+sceptre_user_data:
+  # (Optional) Synapse IDs for owner.txt file
+  SynapseIDs: ["3425758"]  #synapse-service-dian-prod@synapse.org
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "DIAN"
+  # The resource owner
+  OwnerEmail: "synapse-service+dian-prod@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: "true"
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # THIS CURRENTLY ONLY WORKS IN THE us-east-1 REGION!!!!!
+  # Data transfers within the same region between AWS resources are free.
+  # true to restrict downloading files from this bucket to only AWS resources (e.g. EC2 , Lambda) within the same region as this bucket.
+  # This will not allow even the owner of the bucket to download objects in this bucket when not using an AWS resource in the same region!
+  SameRegionResourceAccessToBucket: "true"
```

Comment:
Enabling this only make sense if you `GrantAccess` to users that might download files from regions outside of us-east-1.  Wondering why you are enabling this when the only thing that has access is synapse?